change/@microsoft-teams-js-b469c20c-b2ae-4fcb-aa82-4d19b3292ae9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Replacing prepNextDevRelease.js with prepBetaRelease.js to handle beta version bumping and deprecating use of nextDev tags for releases",
```

Comment:
Super-nit: I'd prefer change file updates to be in past tense. I know this won't be going in the change log because it's of type "none", but others might look at existing change files for examples and I'd like to be as consistent as possible.

Reply:
Thanks, this is good to keep in mind for the future :)